### ISSUE TITLE
[Feature Request] View Output while testing Scripts

### ISSUE THREAD

**Issue by MattisBrumm** [amidaware/tacticalrmm#1631]
**Is your feature request related to a problem? Please describe.**
I had a problem with a Script while I was testing in the Script Manager. To see where the Script is stuck or took so long I decided to put some Write-Hosts in it. 
The Problem is I can only see the Output after the Script is done and while its running there only is the progress circle.

Is somtething planned for that?

**Comment by silversword411** [comment]
No, interesting idea.

Mesh terminal is live, have you tried using that?

One day we might engineer websocket direct connections for like monitoring, this would definitely benefit from something like that.

**Comment by MattisBrumm** [comment]
Yeah i fixed my Script. I just thought it could be an interesting Feature.

In addition this could be interesting for the Checks in the computer too. So you don´t just see Last Output, also you could se Live Output. But this is not necessary more a nice to have. :)
